src/polymer-project.ts

```diff
@@ -112,8 +112,12 @@ export class PolymerProject {
 
   constructor(options?: ProjectOptions) {
     this.root = options.root || process.cwd();
-    this.entrypoint = osPath.resolve(this.root, options.entrypoint);
-    this.shell = osPath.resolve(this.root, options.shell);
+    if (options.entrypoint) {
+      this.entrypoint = osPath.resolve(this.root, options.entrypoint);
+    }
+    if (options.shell) {
+      this.shell = osPath.resolve(this.root, options.shell);
```

Comment:
do we have a preference as a team for using null for explicit "not set" properties?